src/types/user.types.ts

```diff
@@ -0,0 +1,24 @@
+import { Type, Static } from '@sinclair/typebox';
+
+const user = Type.Object({
+  id: Type.Number(),
```

Comment:
Ei ole enään numero vaan uuid
```suggestion
  id: Type.String(),
```